{{cookiecutter.collection_name}}/docs/gen_blocks_catalog.py

```diff
@@ -43,11 +43,13 @@ def insert_blocks_catalog(generated_file):
 
             To register blocks in this module to
             [view and edit them](https://orion-docs.prefect.io/ui/blocks/)
-            on Prefect Cloud:
+            on Prefect Cloud, first [install the required packages](
```

Comment:
```suggestion
            on Prefect Cloud, first [install the required package](
```
are there other packages besides the collection that you'd be referring to here?

Reply:
Yes some Collections require extras like GCP, AWS, dbt in extras